google-oauth-client/src/test/java/com/google/api/client/auth/oauth/OAuthHmacSha256SignerTest.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2012 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.api.client.auth.oauth;
+
+import java.security.GeneralSecurityException;
+import junit.framework.TestCase;
```

Comment:
org.junit preferred

Reply:
The POM file only has the current junit dependency (junit:junit:4.13.2).  I can switch to a different junit, but that would require adding a new dependency.  Which group/artifact/version org.junit package is the preferred one to add and use?

Reply:
junit.* is the old legacy classes. org.junit are the new ones from JUnit 4.

Reply:
Thank you, I misunderstood.  I updated the test to use org.junit / JUnit4